devicetypes/smartthings/cree-bulb.src/cree-bulb.groovy

```diff
@@ -93,7 +93,7 @@ def ping() {
 }
 
 def refresh() {
-    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig() + zigbee.levelConfig()
+    zigbee.onOffRefresh() + zigbee.levelRefresh() + zigbee.onOffConfig(0, 300) + zigbee.levelConfig()
```

Comment:
Need the handling of the configure reporting response instead of generating the checkInterval event in configure